builder/Dockerfile

```diff
@@ -10,6 +10,6 @@ WORKDIR /src
 RUN ["apk", "update"]
 RUN until apk add docker; do :; done
 
-RUN ["cabal", "update"]
+RUN ["stack", "upgrade"]
```

Comment:
Do you consider to nail the stack version to one you explicitly control. So the build is repeatable exactly?